### DISCUSSION TITLE
After migration of Powerapp: Unable to find the default new form for list

### DISCUSSION THREAD

**Discussion by sjorz70** [Zerg00s/FlowPowerAppsMigrator#71] [Q&A] [answered]
I duplicated my Sharepoint within the same tenant with ShareGate to have the copy identical. I downloaded the exported PowerApps and forms and use dthe FlowPowerAppsMigrator tool to update the Source site to the Target without moving lists/libraries. After the migration that showed no errors I imported the PowerApp via make.powerapps.com with Import as New and update the name of the powerapp.

After the import I could see the the app and also from the List Settings I could update the imported PowerApp. 

However, if I click New or Edit a list item, it gives the message "Unable to find the default new form for list for ..."

Anyone who can help me out? it looks like new.aspx and edit.aspx is messed up. We do not have MS SharePoint Designer anymore to update this.

**Comment by Zerg00s** [comment]
Hi @sjorz70, can you delete the problematic list(s) and use ShareGate again. But this time, unselect the "**Customized list forms**" in your configurations section of the copy options screen

**Reply by sjorz70** [reply]
Hello Zerg00, 

After that, I can re-import the converted list right?

**Reply by Zerg00s** [reply]
Yes, you need to re-run the migrator to convert the Power App(s) to be compatible with the new destination list.

**Comment by sjorz70** [comment, marked as answer]
Hi Denis,

We deleted the imported lists and copied them with ShareGate as new lists with the default SharePoint form. Then I converted the flows and apps again and reimported these. That works.

It was a bit trial and error, but the migration tool did some good work replacing the correct URL’s and Form ID’s

Verzonden vanuit Outlook voor iOS<https://aka.ms/o0ukef>
________________________________
Van: Denis Molodtsov ***@***.***>
Verzonden: Tuesday, December 12, 2023 6:54:24 PM
Aan: Zerg00s/FlowPowerAppsMigrator ***@***.***>
CC: Vries, Sjoerd de ***@***.***>; Mention ***@***.***>
Onderwerp: Re: [Zerg00s/FlowPowerAppsMigrator] After migration of Powerapp: Unable to find the default new form for list (Discussion #71)

You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>
Caution: This e-mail originated from outside of Philips, be careful for phishing.


Yes, you need to re-run the migrator to convert the Power App(s) to be compatible with the new destination list.

—
Reply to this email directly, view it on GitHub<https://github.com/Zerg00s/FlowPowerAppsMigrator/discussions/71#discussioncomment-7833758>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BEJ222ULPOO6DY4BRB4EU4DYJCK5BAVCNFSM6AAAAABAP7ES66VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQMZTG42TQ>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

________________________________
The information contained in this message may be confidential and legally protected under applicable law. The message is intended solely for the addressee(s). If you are not the intended recipient, you are hereby notified that any use, forwarding, dissemination, or reproduction of this message is strictly prohibited and may be unlawful. If you are not the intended recipient, please contact the sender by return e-mail and destroy all copies of the original message.

**Comment by sjorz70** [comment]
Thank you for your support. I bought you a coffee from my private account.
